### ISSUE TITLE
Move Accessibility info into dev-wiki and community info

### ISSUE THREAD

**Issue by kprosise** [NetworkGradeLinux/mion-docs#198]
The page about [accessibility](https://docs.mion.io/2021.06-dev/ACCESSIBILITY/) should be removed, with the relevant information being divided: how to use accessibility tools to check documentation should be moved to the wiki, and our pledge/efforts towards accessibility should be moved to the community section.

**Comment by kprosise** [comment]
Closing, will be covered under general/minor improvements.
